python/whylogs/api/store/query.py

```diff
@@ -1,36 +1,26 @@
-from abc import ABC
+from dataclasses import dataclass, field
 from datetime import datetime
-from typing import Optional
+from typing import Optional, Union
 
 
-class BaseQuery(ABC):
-    def __init__(self):
-        self.dataset_id = None
-
-
-class DateQuery(BaseQuery):
-    def __init__(
-        self,
-        dataset_id: str,
-        start_date: datetime,
-        end_date: Optional[datetime] = None,
-        segment_tag: Optional[str] = None,
-    ):
-
-        super().__init__()
-        self.dataset_id = dataset_id
-        self.start_date = start_date
-        self.end_date = end_date
-        self.segment_tag = segment_tag
+@dataclass
+class DateQuery:
+    dataset_id: str
```

Comment:
Meant to remove this since you added the DatasetIdQuery?

Reply:
added an issue for it https://github.com/whylabs/whylogs/issues/1003